Dockerfile

```diff
@@ -11,7 +11,6 @@ RUN apt-get -q update && apt-get -q -y install \
     git \
     curl \
     wget \
-    golang-${GO_VERSION} \
```

Comment:
I think this is a leftover from previous refactors. GO_VERSION is not defined and golang is installed from source below

Reply:
@gshimansky I can leave this change if it does not break anything -- as golang-<empty> will not get installed and it might confuse people or even fail in the future on an apt implementation which fails if the package cannot be found